tests/phpunit/includes/testcase-dominant-color.php

```diff
@@ -0,0 +1,140 @@
+				'expected_color'        => array( 'ff0000' ),
+				'expected_transparency' => false,
+			),
+			'green_gif'     => array(
+				'image_path'            => DIR_TESTDATA . '/images/dominant-color/green.gif',
+				'expected_color'        => array( '00ff00' ),
+				'expected_transparency' => false,
+			),
+			'white_gif'     => array(
+				'image_path'            => DIR_TESTDATA . '/images/dominant-color/white.gif',
+				'expected_color'        => array( 'ffffff' ),
+				'expected_transparency' => false,
+			),
+			'trans_gif'     => array(
+				'image_path'            => DIR_TESTDATA . '/images/dominant-color/trans.gif',
+				'expected_color'        => array( '5a5a5a', '020202' ),
+				'expected_transparency' => true,
+			),
+
+			'red_png'       => array(
+				'image_path'            => DIR_TESTDATA . '/images/dominant-color/red.png',
+				'expected_color'        => array( 'ff0000' ),
+				'expected_transparency' => false,
+			),
+			'green_png'     => array(
+				'image_path'            => DIR_TESTDATA . '/images/dominant-color/green.png',
+				'expected_color'        => array( '00ff00' ),
+				'expected_transparency' => false,
+			),
+			'white_png'     => array(
+				'image_path'            => DIR_TESTDATA . '/images/dominant-color/white.png',
+				'expected_color'        => array( 'ffffff' ),
+				'expected_transparency' => false,
+			),
+			'trans_png'     => array(
+				'image_path'            => DIR_TESTDATA . '/images/dominant-color/trans.png',
+				'expected_color'        => array( '000000' ),
+				'expected_transparency' => true,
+			),
+
+			'red_webp'      => array(
+				'image_path'            => DIR_TESTDATA . '/images/dominant-color/red.webp',
+				'expected_color'        => array( 'ff0000' ),
+				'expected_transparency' => false,
+			),
+			'green_webp'    => array(
+				'image_path'            => DIR_TESTDATA . '/images/dominant-color/green.webp',
+				'expected_color'        => array( '00ff00' ),
+				'expected_transparency' => false,
+			),
+			'white_webp'    => array(
+				'image_path'            => DIR_TESTDATA . '/images/dominant-color/white.webp',
+				'expected_color'        => array( 'ffffff' ),
+				'expected_transparency' => false,
+			),
+			'trans_webp'    => array(
+				'image_path'            => DIR_TESTDATA . '/images/dominant-color/trans.webp',
+				'expected_color'        => array( '000000' ),
+				'expected_transparency' => true,
+			),
+			'balloons_webp' => array(
+				'image_path'            => DIR_TESTDATA . '/images/dominant-color/balloons.webp',
+				'expected_color'        => array( 'c1bbb9', 'c0bab8', 'c3bdbd' ),
+				'expected_transparency' => false,
+			),
+		);
+	}
+
+	/**
+	 * Data provider for test_get_dominant_color_GD.
+	 *
+	 * @return array
+	 */
+	public function provider_get_dominant_color_invalid_images() {
+		$data = array(
+			'tiff' => array(
+				'image_path'            => DIR_TESTDATA . '/images/dominant-color/test-image.tiff',
```

Comment:
Reuse - https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/data/images/test-image.tiff

Reply:
swapped and remove file